boards/arm/olimex_stm32_p103/Kconfig.board

```diff
@@ -0,0 +1,9 @@
+# OLIMEX-STM32-P103 board configuration
+
+# Original work Copyright (c) 2017, Erwin Rol <erwin@erwinrol.com>
+# Modified work Copyright 2020, Sturla Lange
```

Comment:
Please don't retain previous copyright for this kind of board files.
Indeed, what you pointed out as "original" was it self a copy of previous work ...
All these board files are the same and copied from each others.
I prefer to retain the copyright of the one who did the effort of porting and testing the board, instead of having a long list of copyrights...